_docs/release-notes.md

```diff
@@ -24,6 +24,50 @@ Visit [https://app.tidepool.org](https://app.tidepool.org) to see the results of
 
 Don't have a Tidepool account? Visit [tidepool.org/signup](https://tidepool.org/signup) to create your free Tidepool account.  
 
+### 1.17.0 (Released 2019-03-05)
+There was a data paging bug that was triggered in instances where a large gap of time without data is followed by a data point that is diabetes-specific (like a data point) rather than device-related (like an insulin pump event). In such cases, data that occurs before the gap would not be segmented and loaded (we call this paging), but instead would be fetched and processed all at once. Depending on how much data exists before the gap, the browser will either slow or crash. Which is a bummer, so we fixed that.  
+[Feature Requirements](https://trello.com/c/6tfqWKVM/6-v1170-some-user-accounts-processing-all-fetched-data-at-once)
+
+Carb events not associated with a bolus are now displayed in Tidepool Web. Here's looking at you, people entering carbs as Dexcom events or folks uploading carb data via Apple Health.  
```

Comment:
At the moment carb data from devices uploaded with the uploader will not be displayed, as the work to ingest carb events, notes etc. is still to come. So it is only for carb data coming in through Tidepool Mobile at this stage.

Reply:
I think I got over-ambitious because the Trello card mentioned the Dexcom API. This will be revised in the updated PR.

Reply:
I'm sorry, you're right! If coming through the Dexcom API the carb events will be recorded. So how do we make it clear that it works except when you're using the Uploader (for now)?

Reply:
A quick mention should do the trick. Check the latest edit.